engine/renderer/private/gbuffers.cpp

```diff
@@ -15,10 +15,15 @@ GBuffers::GBuffers(const std::shared_ptr<GraphicsContext>& context, glm::uvec2 s
         vk::ImageTiling::eOptimal,
         vk::FormatFeatureFlagBits::eDepthStencilAttachment);
 
-    assert(supportedDepthFormat.has_value() && "No supported depth format!");
-
-    _depthFormat = supportedDepthFormat.value();
-    _shadowFormat = supportedDepthFormat.value();
+    if (supportedDepthFormat.has_value())
+    {
+        _depthFormat = supportedDepthFormat.value();
+        _shadowFormat = supportedDepthFormat.value();
+    }
+    else
+    {
+        assert(false && "No supported depth format!");
+    }
```

Comment:
Why change this to an if statement when the assert exits the program anyway when there is no value available? Couldn't this just have been left as previously, like this:
`assert(supportedDepthFormat.has_value() && "No supported depth format!");`

Reply:
code linting fix 😭